com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -564,7 +672,7 @@ public SceneEventProgressStatus UnloadScene(Scene scene)
         /// </summary>
         private void OnClientUnloadScene()
         {
-            var sceneName = GetSceneNameFromNetcodeSceneIndex(SceneEventData.SceneIndex);
+            var sceneName = GetSceneNameFromBuildIndex(SceneEventData.SceneIndex);
```

Comment:
Wouldn't we want instead to just check if `SceneEventData.SceneIndex` is within the bounds of the `ScenesInBuild` list vs. get the string name and compare to ""?

Reply:
....or...I see that we keep re-calculating `GetSceneNameFromBuildIndex(SceneEventData.SceneIndex)` in each of the exceptions below.  I assume we can just re-use this value there?